src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newObject = {};
+  const clearString = sourceString.trim();
```

Comment:
[CODE KNOWLEDGE] - Don't need to make copy before each operation. Moreover, methods like `trim` don't mutate the original string.